cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -450,23 +448,10 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 			DepositAsset { assets: Wild(AllCounted(2)), beneficiary },
 		]);
 
-		let teleport_xcm_on_bh = Xcm(vec![
-			BuyExecution { fees: local_fee_asset.clone(), weight_limit: Unlimited },
-			DepositAsset {
-				assets: Wild(AllCounted(1)),
-				beneficiary:
-					(AccountId32 { id: assethub_sovereign.clone().into(), network: None },).into(),
-			},
-		]);
-
 		let xcms = VersionedXcm::from(Xcm(vec![
 			WithdrawAsset(assets.clone().into()),
+			BurnAsset(local_fee_asset.clone().into()),
```

Comment:
User will burn the fee asset here and teleport it to BH, use that to BuyExecution for the Export message.